threadedcomments/south_migrations/0004_auto__del_index_threadedcomment_tree_path.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+from django.db import connection
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Removing index on 'ThreadedComment', fields ['tree_path']
+        if connection.vendor == 'mysql':
+            db.execute('DROP INDEX threadedcomments_comment_tree_path ON threadedcomments_comment')
+        else:
+            db.delete_index('threadedcomments_comment', ['tree_path'])
```

Comment:
@vdboor  because if the `connection.vendor` is `mysql` the index was created using raw sql for supporting mysql .. but since the fix is to drop the index we need to drop it using raw sql because south generates an index name with a hash in it.